package.json

```diff
@@ -38,11 +38,11 @@
     "webpack": "^4.0.0 || ^5.0.0"
   },
   "dependencies": {
-    "mem": "^6.1.1",
+    "mem": "^7.0.0-1",
```

Comment:
I think it is not safe to use it right now, it is not stable

Reply:
it's actually the latest stable. `npm dist-tags mem`:
```
latest: 7.0.0-1
next: 7.0.0-3
```

I can downgrade if you want to wait for a proper semver-ed version.

Reply:
Yep, I think it is mistake